internal/compile/manager.go

```diff
@@ -93,7 +93,7 @@ func (c *Manager) processUpdateQueue(ctx context.Context) {
 			case storage.EventReload:
 				c.log.Info("Purging compile cache")
 				c.cache.Purge()
-			case storage.EventAddOrUpdatePolicy, storage.EventDeletePolicy:
+			case storage.EventAddOrUpdatePolicy, storage.EventDeleteDisableOrEnablePolicy:
```

Comment:
It should be just `EventDeleteOrDisablePolicy`. Enabling a policy works differently. I think that's covered by `AddOrUpdatePolicy`.